python/hail/methods/qc.py

```diff
@@ -29,58 +30,43 @@ def sample_qc(dataset, name='sample_qc') -> MatrixTable:
     This method computes summary statistics per sample from a genetic matrix and stores the results as
     a new column-indexed field in the matrix, named based on the `name` parameter.
```

Comment:
I think this could be clearer that a new column-indexed *struct* field is added, with fields as defined below. On my first reading, it wasn't immediately obvious that `dp_stats` is a field in `sample_qc`, not a separate top-level field.